UnitTests/View/Layout/LayoutTests.cs

```diff
@@ -43,7 +43,9 @@ public void TopologicalSort_Recursive_Ref ()
 			var sub2 = new View ();
 			root.Add (sub2);
 			sub2.Width = Dim.Width (sub2);
-			Assert.Throws<InvalidOperationException> (() => root.LayoutSubviews ());
+
+			var exception = Record.Exception (root.LayoutSubviews);
```

Comment:
I've found debugging unit tests that use `Record.Exception` is extra hard. 

Is there a way to refactor tests like this to not use it?

Reply:
@tig I haven't any problem on debugging this. I put a breakpoint on it and press F11 and I can go smoothly into the `LayoutSubviews` method to debugging. For me when I want to prove that something won't throw an exception, I like to use the `Record.Exception`.